reg_tests/check_norms.py

```diff
@@ -54,6 +58,10 @@ def generate_test_norms(testname):
     cmdline = """awk '/Mean System Norm:/ { print $4, $5, $6; }' %s > %s """%(
         logname, norm_name)
     os.system(cmdline)
+    args = parse_arguments()
+    if (args.save_norm_file != None):
```

Comment:
This looks like it will copy the norms before it checks that the test actually passed. Is that what was intended? Seems to me that the desired behavior would be to skip touching gold norms on each ctest run, at least making sure that it is not updated for failing tests.

Reply:
OK. So do I understand correctly that this breaks the current `NALU_GOLD_NORMS_DIR` developer workflow? The checks will always compare against the reference golds?